### PR TITLE
i18n(zh-cn): Update vercel.mdx

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Update vercel.mdx

#### Related issues & labels (optional)

- #6786

### CODE REVIEW

**Comment by liruifengv** [comment]
We have a new commit for `vercel.mdx`, can you update it? https://github.com/withastro/docs/commit/44c3650f84cb8f82454d13bf4d037f77251f1715

**Comment by huyikai** [comment]
> We have a new commit for `vercel.mdx`, can you update it? [44c3650](https://github.com/withastro/docs/commit/44c3650f84cb8f82454d13bf4d037f77251f1715)

no problem

File: src/content/docs/zh-cn/guides/integrations-guide/vercel.mdx [hunk 1]
```diff
@@ -197,7 +197,7 @@ import astroLogo from '../assets/logo.png';
 />
 ```
 
-### devImageService
+### `devImageService`
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/vercel.mdx:200]: 下面的 'sharp' 应该更新成 `sharp`，见：https://github.com/withastro/docs/commit/29dfc8b0679d00fe78f10c43130ffd8f9c0a44c5#diff-71dd54e100b0b6416f7750b7ef6870f8c5b28addb788fd1405d930e14abf820eR205

**Review by liruifengv — APPROVED** [review]
LGTM

**Pull request merged**